samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/servlet/event/ExportEventLogServlet.java

```diff
@@ -0,0 +1,128 @@
+package org.sakaiproject.tool.assessment.ui.servlet.event;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.ResourceBundle;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.apache.commons.text.translate.CsvTranslators.CsvEscaper;
+import org.sakaiproject.component.cover.ComponentManager;
+import org.sakaiproject.samigo.util.SamigoConstants;
+import org.sakaiproject.time.api.UserTimeService;
+import org.sakaiproject.tool.assessment.data.dao.assessment.EventLogData;
+import org.sakaiproject.tool.assessment.services.assessment.EventLogService;
+import org.sakaiproject.tool.assessment.ui.servlet.SamigoBaseServlet;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ExportEventLogServlet extends SamigoBaseServlet {
+
+
+    private static final String CSV_SEPARATOR = ",";
```

Comment:
I feel this is hardcoded, in Spain we use the ";" as delimiter.

Maybe something like this.....

https://source.sakaiproject.org/release/22.1/apidocs/org/sakaiproject/util/FormattedText.html#getDecimalSeparator()

String decimalSeparator = FormattedText.getDecimalSeparator();
String csvSeparator = StringUtils.equals(".", decimalSeparator) ? "," : ";" ;